src/resolvers/resolver.cr

```diff
@@ -44,7 +44,7 @@ module Shards
       end
     end
 
-    protected def install_path
+    def install_path
```

Comment:
Good to know that you don't guarantee _the internal API to be stable_ for you or for those who one day would contribute.

Like I said: it is up to you.